index.html

```diff
@@ -109,10 +109,11 @@ <h3>
                         <span class="carrot"></span>
                         <div class="device-box-header">HTC Vive</div>
                         <span>
-                            Works with <a href="https://webvr.rocks/firefox">Firefox Nightly</a>,
-                            <a href="https://blog.mozvr.com/webvr-servo-architecture-and-latency-optimizations/">Servo</a>, and
+                            Works with <a href="https://webvr.rocks/firefox">Firefox</a>,
+                            <a href="https://blog.mozvr.com/webvr-servo-architecture-and-latency-optimizations/">Servo</a> and
```

Comment:
might as well change this link to https://webvr.rocks/servo or https://blog.servo.org/2018/03/09/servo-and-mixed-reality/

Reply:
Done thanks!